app/cdap/components/GridTable/index.tsx

```diff
@@ -51,6 +53,17 @@ export default function GridTable() {
       count: '0',
     },
   ]);
+  const [toaster, setToaster] = useState<ISnackbarToast>({
+    open: false,
+    isSuccess: false,
+  });
+  const handleSnackbar = () => {
```

Comment:
can we rename this function to specify that it is a success snackbar as we are just setting the success message here. or see if we can handle both success and failure with this single function, in such case, plz send success / failure states as args